apps/app/src/app/pages/(components)/components/(sonner)/sonner.preview.ts

```diff
@@ -25,7 +25,7 @@ export class SonnerPreview {
 
 export const defaultImports = `
 import { toast } from 'ngx-sonner';
-import { HlmToaster } from '@spartan-ng/helm/sonner';
+import { HlmSonnerImports } from '@spartan-ng/helm/sonner';
```

Comment:
Names seem to be mismatched:
```suggestion
import { HlmToasterImports } from '@spartan-ng/helm/sonner';
```